tekton/images/test-runner/Dockerfile

```diff
@@ -215,9 +181,13 @@ RUN gem install mdl -v 0.5.0
 RUN python -m pip install yamllint
 
 # Temporarily add test-infra to the image to build custom tools
-RUN mkdir -p /go/src/github.com/knative/ && git clone https://github.com/knative/test-infra.git /go/src/github.com/knative/test-infra && \
-    make -C /go/src/github.com/knative/test-infra/tools/githubhelper && \
-    cp /go/src/github.com/knative/test-infra/tools/githubhelper/githubhelper /usr/local/bin && \
+RUN mkdir -p /${GOPATH}/src/github.com/knative/ && git clone https://github.com/knative/test-infra.git /${GOPATH}/src/github.com/knative/test-infra && \
```

Comment:
We could build this is a separate container like we do for `kubetest` ?

Reply:
indeed :wink: